Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -432,13 +485,30 @@ private void AddCargoContents(CargoShuttleComponent shuttle, StationCargoOrderDa
         }
     }
 
+    private void OnPalletSale(EntityUid uid, CargoPalletConsoleComponent component, CargoPalletSellMessage args)
+    {
+        var player = args.Session.AttachedEntity;
+
+        if (player == null)
+            return;
+
+        if (!TryComp<TransformComponent>(uid, out var xform) || xform.GridUid == null)
```

Comment:
Transform() proxy method (cast by pattern match here so you dont have to re-declare a griduid local var) (also grey the button if this condition fails)